docs/farming-&-staking/farming/advanced-cli/cli-install.mdx

```diff
@@ -33,8 +33,13 @@ There are multiple versions of our releases available below.
 
 ## Get Started with Advanced CLI Installation
 
+:::note
+Please note that a wallet with a Substrate address is required to run the software.<br />
+To learn how to create a Substrate wallet, please visit our pages on [SubWallet (recommended)](versioned_docs/version-latest/farming-&-staking/wallets/subwallet.md) or [Polkadot.js](versioned_docs/version-latest/farming-&-staking/wallets/polkadot.md).
```

Comment:
This is not critical since we're likely removing the pre-release docs, but the links should lead to just /docs instead of /versioned_docs. They were correct previously.